README.md

```diff
@@ -5,8 +5,9 @@ when they want to switch places in kindergartens.
 
 There are currently no vacancies, so when they want to move, they get an answer from the authorities: all places are filled.
```

Comment:
Approved

Reply:
Please click on `Approve` button... it is disabled for me
![Screenshot 2023-05-08 at 20 31 50](https://user-images.githubusercontent.com/1426092/236903188-43cf71b4-7489-4954-9d0d-6738f1c9bdca.png)